papers/api.py

```diff
@@ -213,7 +213,11 @@ def api_paper_query(request):
     except ValueError:
         raise BadRequest('Invalid paper')
 
-    return {'status': 'ok', 'paper': p.json()}
+    try:
```

Comment:
The `BarePaper.create` call above is only used to compute fingerprint? Or is any side effect expected? If this is the case, it could probably use a comment :)

Reply:
Done!